quantum/keymap_extras/keymap_swedish_osx_ansi.h

```diff
@@ -0,0 +1,233 @@
+#define SE_EXLM S(SE_1)    // ! 
+#define SE_DQUO S(SE_2)    // "
+#define SE_HASH S(SE_3)    // #
+#define SE_CURR S(SE_4)    // €
+#define SE_PERC S(SE_5)    // %
+#define SE_AMPR S(SE_6)    // &
+#define SE_SLSH S(SE_7)    // /
+#define SE_LPRN S(SE_8)    // (
+#define SE_RPRN S(SE_9)    // )
+#define SE_EQL  S(SE_0)    // =
+#define SE_QUES S(SE_PLUS) // ?
+#define SE_GRV  S(SE_ACUT) // `
+// Row 2
+#define SE_CIRC S(SE_DIAE) // ^ (dead)
+#define SE_ASTR S(SE_QUOT) // *
+// Row 4
+#define SE_SCLN S(SE_COMM) // ;
+#define SE_COLN S(SE_DOT)  // :
+#define SE_UNDS S(SE_MINS) // _
+
+/* Alted symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬─────┐
+ * │ ≤ │ © │ @ │ £ │ $ │ ∞ │   │ | │ [ │ ] │ ≈ │ ± │   │     │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬───┤
+ * │     │ • │ Ω │ é │ ® │ † │ µ │ ü │ ı │ œ │ π │ ˙ │ ~ │ ™ │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤
+ * │      │  │ ß │ ∂ │ ƒ │ ¸ │ ˛ │ √ │ ª │ ﬁ │ ø │ æ │      │
+ * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──────┤
+ * │        │ ÷ │   │ ç │ ‹ │ › │ ‘ │ ’ │ ‚ │ … │ – │        │
+ * ├─────┬──┴─┬─┴───┼───┴───┴───┴───┴───┴───┼───┴─┬─┴──┬─────┤
+ * │     │    │     │                       │     │    │     │
+ * └─────┴────┴─────┴───────────────────────┴─────┴────┴─────┘
+ * 
+ */
+// Row 1 
+#define SE_LTEQ A(SE_LABK) // ≤
+#define SE_COPY A(SE_1)    // ©
+#define SE_AT   A(SE_2)    // @
+#define SE_PND  A(SE_3)    // £
+#define SE_DLR  A(SE_4)    // $
+#define SE_INFN A(SE_5)    // ∞
+#define SE_PIPE A(SE_7)    // |
+#define SE_LBRC A(SE_8)    // [
+#define SE_RBRC A(SE_9)    // ]
+#define SE_AEQL A(SE_0)    // ≈
+#define SE_PLMN A(SE_PLUS) // ±
+// Row 2
+#define SE_BULT A(SE_Q)    // •
+#define SE_OMEG A(SE_W)    // Ω
+#define SE_EACU A(SE_E)    // é
+#define SE_REGD A(SE_R)    // ®
+#define SE_DAGG A(SE_T)    // †
+#define SE_MICR A(SE_Y)    // µ
+#define SE_UDIA A(SE_U)    // ü
+#define SE_DLSI A(SE_I)    // ı
+#define SE_OE   A(SE_O)    // œ
+#define SE_PI   A(SE_P)    // π
+#define SE_DOTA A(SE_ARNG) // ˙
+#define SE_TILD A(SE_DIAE) // ~ (dead)
+#define SE_TM   A(SE_QUOT) // ™
+// Row 3
+#define SE_APPL A(SE_A)    //  (Apple logo)
+#define SE_SS   A(SE_S)    // ß
+#define SE_PDIF A(SE_D)    // ∂
+#define SE_FHK  A(SE_F)    // ƒ
+#define SE_CEDL A(SE_G)    // ¸
+#define SE_OGON A(SE_H)    // ˛
+#define SE_SQRT A(SE_J)    // √
+#define SE_FORD A(SE_K)    // ª
+#define SE_FI   A(SE_L)    // ﬁ
+#define SE_OSTR A(SE_ODIA) // ø
+#define SE_AE   A(SE_ADIA) // æ
+// Row 4
+#define SE_DIV  A(SE_Z)    // ÷
+#define SE_CCCE A(SE_C)    // ç
+#define SE_LSAQ A(SE_V)    // ‹
+#define SE_RSAQ A(SE_B)    // ›
+#define SE_LSQU A(SE_N)    // ‘
+#define SE_RSQU A(SE_M)    // ’
+#define SE_SLQU A(SE_COMM) // ‚
+#define SE_ELLP A(SE_DOT)  // …
+#define SE_NDSH A(SE_MINS) // –
+
+/* Shift+Alted symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬─────┐
+ * │ ≥ │ ¡ │   │ ¥ │ ¢ │ ‰ │   │ \ │ { │ } │ ≠ │ ¿ │   │     │
```

Comment:
Looks like there are some missing symbols here? or are they duplicates?
![image](https://user-images.githubusercontent.com/4781841/123678711-0374ba00-d88a-11eb-8f4d-e466678c291f.png)

Reply:
I have pushed a couple of new commits that adds missing keys and I've checked for duplicates. 🙂

Reply:
Yes. The missing ones should all be duplicates